src/assets/Generator.Shared/RequestContentHelper.cs

```diff
@@ -11,6 +11,43 @@ namespace Azure.Core
 {
     internal static class RequestContentHelper
     {
+        public static RequestContent FromEnumerable<T>(IEnumerable<T> enumerable) where T: notnull
```

Comment:
Just curious, will C# compiler be able to differentiate between these two methods?

Reply:
Yes, it chooses the one that fits better.